knowledge-base/how-to-dynamically-style-react-report-viewer.md

```diff
@@ -0,0 +1,81 @@
+---
+title: How to Dynamically Style the React Report Viewer
+description: How to Dynamically Style the React Report Viewer
+type: how-to
+page_title: How to Dynamically Style the React Report Viewer
+slug: how-to-dynamically-style-react-report-viewer
+position: 
+tags: Web, React, ReactJS, Style, Styling
+ticketid: 1578385
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+		<tr>
+			<td>Report Viewer</td>
+			<td>React</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+In certain cases, one might want to change the styling of the React Report Viewer widget dynamically.
+Be it based on user interaction after the report viewer has loaded, or based on specific user selections, or etc., the styles can be controlled
+through the `viewerContainerStyle` property. 
```

Comment:
The styles can be controlled through the `viewerContainerStyle` property even in run-time.